src/celeste/mergeSortedArrays.js

```diff
@@ -0,0 +1,35 @@
+// Merge Sorted Arrays (medium)
+//
+// You are given two arrays of numbers that are already sorted in to
+// ascending order.
+//
+// Write a function called mergeSortedArrays() that takes two sorted arrays
+// and merges them in to one array that is also sorted in ascending order.
+//
+// -----------------------------------------------------------------------
+
+function mergeSortedArrays(a, b) {
```

Comment:
a and b are really generic arguement names. I would go with arrayOne and arrayTwo, or sortedArr1 sortedArr2 or something more descriptive